src/main/webapp/script.js

```diff
@@ -12,19 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-$(document).ready(function() {
-    $('.list').click(function() {
-      const value = $(this).attr('data-filter');
-      if (value == 'all') {
-        $('.itemBox').show('1000');
-      }
-      else {
-        $('.itemBox').not('.'+value).hide('1000');
-        $('.itemBox').filter('.' + value).show('1000');
-      }
+$(document).ready(function () {
+    $('.list').click(function () {
+        const value = $(this).attr('data-filter');
+        if (value == 'all') {
+            $('.itemBox').show('1000');
+        }
+        else {
+            $('.itemBox').not('.' + value).hide('1000');
+            $('.itemBox').filter('.' + value).show('1000');
+        }
     })
-  
-    $('.list').click(function() {
-      $(this).addClass('active').siblings().removeClass('active');
+
+    $('.list').click(function () {
+        $(this).addClass('active').siblings().removeClass('active');
     })
-})
\ No newline at end of file
+})
+
+async function getGuide() {
+    const responseFromServer = await fetch('/guides');
+    const jsonResponse = await responseFromServer.json();
+    const guides_container = document.getElementById($('#product'));
+
+    for (let i = 0; i < jsonResponse.length; i++) {
+        $('#product').append(guidesToHTML(jsonResponse[i]));
+    }
+}
+
+function guidesToHTML(guide) {
+    let id = guide.id;
+    let title = guide.title;
+    let description = guide.description;
+    let timestamp = guide.timestamp; // Returns in ms
+    let date = new Date(timestamp); // Convert to date
+
+    return `<div class="card w-75 itemBox item1">
+        <div class="card-header">${id}</div>
```

Comment:
If you prefer, you can write ${guide.id} in the template and remove the variable declarations above. Writing ${new Date(guide.timestamp).toDateString()} should also work.

Reply:
I removed the variable declarations as it cleans up the code